cdn/dev/keyboard-search/dedicated-landing-pages.js

```diff
@@ -0,0 +1,66 @@
+//
+// Meta search results for dedicated landing pages
+//
+
+const dedicatedLandingPages = [
+  {
+    id: "/amharic",
+    terms: ['amharic', 'ge\'ez', 'geez', 'ethiopic'],
+    description: "Amharic keyboards recommended by the Keyman team",
+    name: "Amharic Keyboards Home Page"
+  },
+  {
+    id: "/burmese",
+    terms: ['burmese', 'myanmar'],
+    description: "Burmese (Myanmar) keyboards recommended by the Keyman team",
+    name: "Burmese Keyboards Home Page"
+  },
+  {
+    id: "/cameroon",
+    terms: ['cameroon'],
+    description: "Cameroon keyboards recommended by the Keyman team",
+    name: "Cameroon Keyboards Home Page"
+  },
+  {
```

Comment:
Are we intentionally not adding an entry to the eurolatin landing page?

Reply:
I missed it initially, but on reflection, I don't think we want it to be in the search results like that. It will come up for many searches anyway, and it's not the most ideal keyboard for most languages.

FWIW, I have #174 open for resolution in 14.0 which is about checking the landing page content and responsive design and we could tweak these search terms further when working on those.